fix: engine warning on console for ui (#403)
verdaccio/ui: README.md

## Contributing

> We use `>=yarn@1.13.0`, keep on mind we use lock file and use at least Node `v10.13.0` to be able to build the project.

Reviewer: ```suggestion
We use `>=yarn@1.13.0`, keep in mind that we use lockfiles and use at least Node `v10.13.0` to be able to build the project.
```
